pkg/networkservice/common/connect/server.go

```diff
@@ -56,8 +56,8 @@ type connectServer struct {
 	ctx                 context.Context
 	dialOptions         []grpc.DialOption
 	clientFactory       func(ctx context.Context, conn grpc.ClientConnInterface) networkservice.NetworkServiceClient
-	clientMap           map[string]*clientEntry // key == url as string
-	connections         map[string]*clientEntry // Connection map is required to close using connection id.
+	clients             map[string]*clientEntry // key == url as string
+	connections         map[string]*url.URL     // Connection map is required to close using connection id.
```

Comment:
This seems to be more of a clientURLs than a connections ...

Reply:
Yes, I'm still not fully sure should it be separate chain element and request clientURI to be part of context always.

Reply:
Say more.